lib/aerospike/cluster.rb

```diff
@@ -118,7 +118,7 @@ def batch_read_node(partition, replica_policy)
         when Aerospike::Replica::RANDOM
           return random_node
         else
-          raise Aerospike::Exceptions::InvalidNode("invalid policy.replica value")
+          raise Aerospike::Exceptions::InvalidNode.new("invalid policy.replica value")
```

Comment:
Probably the most canonical syntax would be:

```
raise ExceptionClass, "Message"
```

However what you merged is also okay.

Reply:
(I would follow whatever is common in the code base already).